functions/packages.bash

```diff
@@ -798,3 +798,138 @@ setup_evcc() {
   echo -n "$(timestamp) [openHABian] Created EVCC config, restarting ... "
   if cond_redirect systemctl restart evcc.service; then echo "OK"; else echo "FAILED"; fi
 }
+
+## Function for (un)installing ESPhome dashboard
+## The function must be invoked UNATTENDED.
+## Valid arguments: "install" or "remove"
+##
+##  install_esphomedashboard(String action)
+##
+##
+install_esphomedashboard() {
+  local port=6052
+  local installText="This will install ESPhome dashboard\nUse the web interface on port $port to access ESPhome dashboard web interface."
+  local removeText="This will remove ESPhome dashboard"
+
+  ESPHOME_DIR="/opt/esphomedashboard"
+  SERVICE_TEMPLATE="../includes/esphome-dashboard.service.template" # Update with the actual path
+
+  if [[ $1 == "remove" ]]; then
+    if [[ -n $INTERACTIVE ]]; then
+      whiptail --title "ESPhome dashboard removal" --msgbox "$removeText" 7 80
+    fi
+    echo "$(timestamp) [openHABian] Starting ESPHome Dashboard uninstallation..."
+
+    # Stop the ESPHome Dashboard service
+    echo "$(timestamp) [openHABian] Stopping the ESPHome Dashboard service..."
+    if ! systemctl stop esphome-dashboard.service; then
+      echo "$(timestamp) [openHABian] Error: Failed to stop ESPHome Dashboard service."
+      return
+    fi
```

Comment:
To be consistent with elsewhere please do something like:

```suggestion
    echo -n "$(timestamp) [openHABian] Stopping the ESPHome Dashboard service... "
    if ! cond_redirect systemctl stop esphome-dashboard.service; then echo "FAILED (stop service)"; return 1; fi
```

Reply:
resolved